test/CrateTest.js

```diff
@@ -3,7 +3,7 @@ var should = require('should'),
   path = require('path'),
   os = require('os'),
   fs = require('fs'),
-  tungus = require('tungus'),
+  //tungus = require('tungus'),
```

Comment:
I had crazy issues once I added my code. For some reason the id would start off as '-2' then switch to '2'. It caused most tests to fail.

Any suggestions?